### DISCUSSION TITLE
[ERROR] [/pkg/scan/job.go:342]: scan job: submit scan request: v1 client: submit scan: performing scan: creating scan job: creating scan job: dial tcp: lookup harbor-harbor-redis on 127.0.0.11:53: server misbehaving: general response handler: unexpected status code: 500, expected: 202

### DISCUSSION THREAD

**Discussion by asd99557** [goharbor/harbor#15809] [General]
After install latest version of harbor an integration with clair given below error.
[ERROR] [/pkg/scan/job.go:342]: scan job: submit scan request: v1 client: submit scan: performing scan: creating scan job: creating scan job: dial tcp: lookup harbor-harbor-redis on 127.0.0.11:53: server misbehaving: general response handler: unexpected status code: 500, expected: 202

**Comment by heww** [comment]
Looks like this is the DNS problem of the env, could you try to test the trivy adapter to check whether this problem also exists?

BTW, the Clair scanner is not maintained by the harbor team now.
